lib/model/index.js

```diff
@@ -105,6 +105,16 @@ export function Base() {
 export function Shape() {
   Base.call(this);
 
+  /**
+   * Indicates frame shapes
+   *
+   * @name Shape#frameOnly
+   * @type Boolean
+   */
+  Object.defineProperty(this, 'frameOnly', {
```

Comment:
If we `Object#defineProperty` here the property won't be enumerable (and JSON#stringifiable). I'd suggest we do not define the property but keep the JSDoc to indicate that shape property.